app/javascript/orangelight/availability.es6

```diff
@@ -321,15 +324,10 @@ export default class AvailabilityUpdater {
       display_request = 'true';
     }
     // if it's on CDL then it can't be requested
-    if (availability_label_text === "Reserved for digital lending") {
+    if (cdl) {
       location_services_element.remove();
     }
-    if (availability_info['on_reserve'] === 'Y') {
-      return location_services_element.remove();
-    }
-    if (availability_info['status_label'].toLowerCase() === 'unavailable') {
```

Comment:
Patrons can request 'Unavailable'. Based on the status we should not update the data-requestable to false. Confirm with @kevinreiss and @carolyncole